app/src/ui/lib/conflicts/unmerged-file.tsx

```diff
@@ -47,7 +47,7 @@ export const renderUnmergedFile: React.SFC<{
    */
   readonly manualResolution?: ManualConflictResolution
   /** branch we were on before starting merge */
-  readonly ourBranch: string
+  readonly ourBranch?: string
```

Comment:
📝 A bit more context on this prop would be helpful. What does passing `undefined`  mean (i.e., why would we want to pass `undefined` to this component instead of the branch name)?

Reply:
Added in caa88e62e8351702e520bd22c71c88b6c30fd6c7 - let me know if there's anything else I can clarify while I'm in here